components/x-live-blog-post/src/LiveBlogPost.jsx

```diff
@@ -47,7 +47,7 @@ const LiveBlogPost = (props) => {
 				<button
 					onClick={backToTop}
 					aria-labelledby="Back to top"
-					className={styles['live-blog-post-controls__back-to-top-button']}
+					className={`live-blog-post-controls__back-to-top-button ${styles['live-blog-post-controls__back-to-top-button']}`}
```

Comment:
**Question:** Have you added it in this way so a consumer could decide to override the `display:none` if they wanted to?

Reply:
Yes, just exposing that option too.

Reply:
Ok, you know what comments coming next 😉 is there a requirement for this? the reason I ask is that if I saw this code in the future I wouldn't have a clue why we are applying the same class names twice. This could be solved with a code comment but if theres no need for it right now I would lean towards not doing it until its needed